LICENSES/README.md

```diff
@@ -13,7 +13,14 @@ The 'LICENSES/' directory contains all the licenses used by the sources included
 the systemd project source tree.
 
 Unless otherwise noted, the systemd project sources are licensed under the terms
-and conditions of the **GNU Lesser General Public License v2.1 or later**.
+and conditions of
+**LGPL-2.0-or-later** (**GNU Lesser General Public License v2.1 or later**).
```

Comment:
`LGPL-2.1-or-later`? here and below

Reply:
actually we have one file in basic and one in shared that are `LGPL-2.0-or-later`

Reply:
but they are or-later, so the end result is 2.1